dev/reference/sql/partitioning.md

```diff
@@ -0,0 +1,698 @@
+---
+title: 分区表
+category: reference
+aliases: ['/docs-cn/sql/partitioning/']
+---
+
+# 分区类型
```

Comment:
Please use a sentence or two to introduce what we're going to cover in this document. This helps users decide whether they should read it.